fighthealthinsurance/chat_interface.py

```
@@ -1206,34 +1197,60 @@ async def search_and_store_pubmed():
+                        else:
+                            summarized_context = (
+                                f"Earlier conversation summary: {history_summary}"
+                            )
+
+                        # Store the summary for future calls
+                        if not chat.summary_for_next_call:
+                            chat.summary_for_next_call = []
+                        chat.summary_for_next_call.append(summarized_context)
+
+                        logger.info(
+                            f"Summarized {len(messages_to_drop)} messages for chat {chat.id}"
```

Ruff (0.14.10)

Undefined name `messages_to_drop`

(F821)